source/documentation/monitoring_apps/metrics.md

```diff
@@ -2,110 +2,95 @@
 
 ## Metrics
 
-Cloud Foundry provides time series data, or metrics, for each instance of your PaaS app. You can receive, store and view this data in a monitoring system of your choice by either:
+Cloud Foundry provides time series data known as metrics for each instance of your PaaS app. You can receive, store and view this data in a monitoring system of your choice by deploying either the:
 
-- using the [Prometheus](https://prometheus.io/) [external link] endpoint provided by the GOV.UK PaaS team
-- deploying the `paas-metric-exporter` app to push metrics data in [StatsD](https://github.com/etsy/statsd/wiki) [external link] format
+- [`paas-prometheus-exporter`](https://github.com/alphagov/paas-prometheus-exporter) app to collect and display metrics in the [Prometheus format](https://prometheus.io/docs/introduction/overview) [external links]
+- [`paas-metric-exporter`](https://github.com/alphagov/paas-metric-exporter) app to display metrics data in the [StatsD format](https://github.com/etsy/statsd/wiki) [external links]
```

Comment:
I'm late to the party, but using `display` here is wrong. The paas-metric-exporter doesn't have an API endpoint and it pushes metrics regularly to a StasdD server, so the original sentence was correct.